extensions/amp-bind/0.1/amp-state.js

```diff
@@ -43,7 +43,8 @@ export class AmpState extends AMP.BaseElement {
   activate(invocation) {
     const event = invocation.event;
     if (event && event.detail) {
-      this.updateState_(event.detail.response, /* isInit */ false);
+      this.updateState_(event.detail.response,
```

Comment:
Thanks for the explanation. I'm thinking that removing this action might be a good idea for both the changes introduced in this PR and for developers who use bind. It looks like the action could be completely replaced with `AMP.setState({myAmpState: event.detail.response})` which gives the following advantages

1. We won't have two ways of doing the same thing
2. We can be entirely consistent about updates to `amp-state`. Any direct update on the `amp-state` (like when its src attribute changes) will not affect other `amp-state` elements until the next evaluation (that isn't caused by an `amp-state` update). 
3. We can assume that any change on an `amp-state` element after init has been caused by an XHR update, allowing us to revert most of the changes to this file. `updateState` would just need to pass `!isInit` through to `setState` as its third argument.
4. The third argument to `setState` can then really be true if the `setState` comes from an `amp-state` and false otherwise. Presently the param is true if we believe the state update could cause a loop, and false otherwise.

WDYT?